kafka-ui-api/src/main/java/com/provectus/kafka/ui/cluster/service/MetricsUpdateService.java

```diff
@@ -16,10 +18,11 @@
     private final KafkaService kafkaService;
     private final ZookeeperService zookeeperService;
 
-    @Async
     public void updateMetrics(KafkaCluster kafkaCluster) {
-        log.debug("Start getting metrics for kafkaCluster: " + kafkaCluster.getName());
-        kafkaService.loadClusterMetrics(kafkaCluster);
-        zookeeperService.checkZookeeperStatus(kafkaCluster);
+        Mono.just(true)
```

Comment:
looks strange, is it possible to avoid Mono.just here?

Reply:
removed mono from this method